schemas/core/components/travel_mode.json

```diff
@@ -1,27 +1,24 @@
 {
```

Comment:
It looks we use nearly all possible file naming styles: camelCase, e.g.`pointCost.json`, underscore `travel_mode.json`. dash `iot-thing-shadow.json`, underscore upper case `MODE_FERRY.json`

For consistency It would be good to settle on one. The most common in ecosystem (I see it as a standard) is lowercase and dash separation, so `point-cost.json`, `travel-model.json`, `iot-thing-shadow.json`, `mode-ferry.json`. I would recommend settling on that.

Reply:
@medikoo i'll amend the names :)

Reply:
The MODE_CAR /etc reflect the name oi the properties in the data. Don't know if we want to preserve those, so it's clear what they refer to?

Reply:
@konker I think it's still better to keep all lower case. Generally sticking to *one* case (instead of mixed case) is good rule to avoid issues with case sensitive/insensitive systems (works on ones, doesn't work on others).